packs/equipment/chimera-flail.json

```diff
@@ -51,7 +51,224 @@
+                "option": "chimera-flail:charging-horns-used",
+                "predicate": [
+                    "chimera-flail:head:goat"
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "DamageDice",
+                "override": {
+                    "damageType": "piercing"
+                },
+                "predicate": [
+                    {
+                        "or": [
+                            "chimera-flail:head:dragon",
+                            "chimera-flail:head:lion"
+                        ]
+                    }
+                ],
+                "selector": "{item|id}-damage"
+            },
+            {
+                "damageType": "{item|flags.pf2e.rulesSelections.dragonHeadDamage}",
+                "key": "FlatModifier",
+                "predicate": [
+                    "chimera-flail:head:dragon",
+                    {
+                        "not": "chimera-flail:bursting-breath-used"
+                    }
+                ],
+                "selector": "{item|id}-damage",
+                "value": 1
+            },
+            {
+                "damageCategory": "precision",
+                "key": "FlatModifier",
+                "predicate": [
+                    "chimera-flail:head:lion",
+                    "target:condition:off-guard",
+                    {
+                        "nor": [
+                            "chimera-flail:pouncing-whirl",
+                            "chimera-flail:pouncing-whirl-used"
+                        ]
+                    }
+                ],
+                "selector": "{item|id}-damage",
+                "value": 1
+            },
+            {
+                "category": "precision",
+                "diceNumber": 1,
+                "dieSize": "d4",
+                "key": "DamageDice",
+                "predicate": [
+                    "chimera-flail:head:lion",
+                    "chimera-flail:pouncing-whirl",
+                    "target:condition:off-guard"
+                ],
+                "selector": "{item|id}-damage"
+            },
+            {
+                "damageType": "force",
+                "diceNumber": 1,
+                "dieSize": "d4",
+                "key": "DamageDice",
+                "predicate": [
+                    "chimera-flail:head:goat",
+                    "chimera-flail:stride-half",
+                    {
+                        "not": "chimera-flail:charging-horns-used"
+                    }
+                ],
+                "selector": "{item|id}-damage"
+            },
+            {
+                "damageType": "force",
+                "diceNumber": 1,
+                "dieSize": "d6",
+                "key": "DamageDice",
+                "predicate": [
+                    "chimera-flail:head:goat",
+                    "chimera-flail:charging-horns"
```

Comment:
The text didn't say it was exclusive of the "Stride at least half your Speed" damage bonus.